src/authentication/components/ExecutionConfigModal.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useForm } from "react-hook-form";
+import {
+  ActionGroup,
+  AlertVariant,
+  Button,
+  ButtonVariant,
+  Form,
+  FormGroup,
+  Modal,
+  ModalVariant,
+  TextInput,
+  ValidatedOptions,
+} from "@patternfly/react-core";
+import { CogIcon, TrashIcon } from "@patternfly/react-icons";
+
+import type AuthenticatorConfigRepresentation from "keycloak-admin/lib/defs/authenticatorConfigRepresentation";
+import type AuthenticatorConfigInfoRepresentation from "keycloak-admin/lib/defs/authenticatorConfigInfoRepresentation";
+import type { ConfigPropertyRepresentation } from "keycloak-admin/lib/defs/authenticatorConfigInfoRepresentation";
+import type { ExpandableExecution } from "../execution-model";
+import { useAdminClient, useFetch } from "../../context/auth/AdminClient";
+import { useAlerts } from "../../components/alert/Alerts";
+import { HelpItem } from "../../components/help-enabler/HelpItem";
+
+export const ExecutionConfigModal = ({
+  execution,
+}: {
+  execution: ExpandableExecution;
+}) => {
+  const { t } = useTranslation("authentication");
+  const adminClient = useAdminClient();
+  const { addAlert, addError } = useAlerts();
+
+  const [show, setShow] = useState(false);
+  const [config, setConfig] = useState<AuthenticatorConfigRepresentation>();
+  const [configDescription, setConfigDescription] =
+    useState<AuthenticatorConfigInfoRepresentation>();
+
+  const { register, errors, setValue, handleSubmit } = useForm();
+
+  const setupForm = (
+    configDescription: any,
+    config?: AuthenticatorConfigRepresentation
+  ) => {
+    configDescription.properties.map(
```

Comment:
This `map()` operation does not actually get assigned to anything, perhaps you meant to do a `forEach()`?
```suggestion
    configDescription.properties.forEach(
```